framework/components/ecl-expandables/expandables.js

```diff
@@ -0,0 +1,15 @@
+export const toggleExandable = toggleElement => {
```

Comment:
Regarding this function: either the element to toggle exists, or it doesn't. I'll add a check but no default.

Besides that, we could maybe add another function which, on "DOMContentLoaded" event, will parse the DOM and automatically attach the click event listener to all the expandables links. It will only be a helper and will still have to be called by the implementer.